gatsby-config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  siteMetadata: {
+    title: 'TC39',
+    lang: 'en-US',
```

Comment:
That reminds me that we didn't talk about the website translations. Do we even need them?

Reply:
Translations would be great! Although it would be impractical to interact with TC39 in other languages, a translated introduction for generalist audiences would be really helpful. We have a big US-centric bias.

Reply:
There are great solutions for multi-lingual. I think it's important but maybe not relevant right now as we are deploying an MVP